Error.name的六种对应的信息
1.EvalError: eval()的使用和定义不一致
2.RangeError： 数值越界
3.ReferenceError：非法或不能识别的引用数值
4.SyntaxError：发生语法解析错误
5.TypeError：操作数类型错误
6.URLError：URL处理函数使用不当


try {
    console.log("a");
    console.log(b);
    console.log("c");
} catch (e) {
         console.log(e.name + e.message);
    }
    console.log("d");

try里面的代码报错的时候,catch里面的代码才会执行,finally里面的代码永远会执行
　　catch和finally里面,正常的代码会从上到下顺序执行
　　如果只是catch里面代码出错,则报catch里面的错误
　　如果catch和finally都出错则会报finally里面的错误